package.json

```diff
@@ -3,7 +3,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "vite build",
+    "build": "NODE_OPTIONS=--max-old-space-size=16384 vite build",
```

Comment:
We should remove this from the github action if we're putting it in here

Reply:
Removed these in 9c8692d